go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/ipfs/kubo
 
+replace github.com/libp2p/go-libp2p-pubsub => github.com/smrz2001/go-libp2p-pubsub v0.0.0-20230110155724-04bfcf58514f
```

Comment:
FYSA I've pushed this change, so we can see if CI passes.
(It is late Friday so will do proper review on Monday)

@smrz2001 In the future, you can point at an arbitrary commit from your own fork via:

```console
$  go mod edit -replace github.com/libp2p/go-libp2p-pubsub=github.com/smrz2001/go-libp2p-pubsub@04bfcf58514f59bddb650fec5b2edac018c8c406
```

Reply:
Thanks, so much, @lidel!! I didn't know we could point to fork commits like that!

Reply:
I'll fix any CI issues that come up.